nmdc_server/auth.py

```diff
@@ -51,11 +60,14 @@ class Token(BaseModel):
 
 
 async def get_token(
-    request: Request, access_token: Dict[str, Any] = Security(oauth2_scheme)
```

Comment:
I'm hesitant to remove this line. Do you know what `Security(oauth2_schema)` does here?

Reply:
Thanks for point that out. Some of this is still a little opaque to me, but after some additional testing it seems like the additional scheme that I added in my original changes wasn't necessary. So now the only change to this line is correcting the type annotation.